Fix SCPD SASS imports

stvstnfrd opened the pull request:
by symlinking to Lagunita.
That's what we were already doing with SUclass and CME.

Without this, static asset compilation breaks under Hawthorn
when the imported pattern-library file cannot be found.

This change should be backward-compatible.

stvstnfrd committed:
Fix SCPD SASS imports

by symlinking to Lagunita.
That's what we were already doing with SUclass and CME.

Without this, static asset compilation breaks under Hawthorn
when the imported pattern-library file cannot be found.

This change should be backward-compatible.

caesar2164 reviewed (changes requested):
Symlink all the things?

stvstnfrd committed:
TODO: squash

stvstnfrd committed:
Merge branch 'master' into hawthorn/wip

stvstnfrd committed:
Stop using platform mixin for Stanford font load

As the base implementation expects to find `*.eot` files, which the
SourceSans font does not support via Google Fonts (from where we load
our web fonts).

stvstnfrd committed:
Symlink core SCPD SASS files

stvstnfrd committed:
rename scpd file

stvstnfrd committed:
try-fontface

stvstnfrd committed:
Fix header

stvstnfrd committed:
Allow SU header/footer to fill width

stvstnfrd committed:
fix header

stvstnfrd committed:
su-header-margin-padding

stvstnfrd commented:
@caesar2164 closing in favor of https://github.com/Stanford-Online/openedx-themes/pull/80
